docs/coding-guidelines/css.md

```diff
@@ -98,52 +98,9 @@ Under the hood, we are using webpack and its `sass-loader`, for compiling the st
 
 ## Media Queries
 
-We don't do device specific breakpoints, we use layout-specific breakpoints that communicate the state of the UI. DO NOT define your own media queries. Utilize the mixins [provided in `_mixins.scss`](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_breakpoints.scss). Furthermore, we are pushing for a mobile-first approach to media queries, meaning your default styles should apply to mobile, and desktop should build on top of that. This means that devices with smaller screens have to process less CSS (which makes sense since they are generally less powerful). You should avoid the use of `<` breakpoints like this:
```

Comment:
I think this some of this documentation might actually still be useful, especially for devs who don't have extensive browser-client experience. Do you think we could just translate the uses of `breakpoint` to `break-*` instead of removing it completely? Or else, what's the reasoning for removing it entirely?

As an aside, it's nice that we're removing some room for variations in how breakpoints are applied given that the GB breakpoints are more restrictive (doesn't look like there's a `<` vs `>` problem); however, I think the approach documented in the bad/good section below is still valid.